bower.json

```diff
@@ -5,7 +5,7 @@
   "dependencies": {
     "polymer": "Polymer/polymer#^1.9.0",
     "iron-image": "^2.1.2",
-    "paper-tooltip": "^2.0.1",
+    "kwc-paper-tooltip": "git@github.com:KanoComponents/kwc-paper-tooltip#b8ba64d13701952a8d8d7a82e43327a3afd4f5c5",
```

Comment:
Do you think we could perhaps make a release and refer to the kwc-paper-tooltip version?

Reply:
Yes, it's want I would like to do.
I think it's the best solution.
Can I make a release?

Reply:
Yes!!! Go for it! Since your [PR](https://github.com/KanoComponents/kwc-paper-tooltip/pull/2) got merged and because it contains a new feature, you should definitely create a release for it! 

Some of the `kwc` components don't have releases but it's because they might be lacking documentation/tests (please don't make a release on those). In general all our `kwc` components have versions and we should make a release when we merge a PR with a feature! We follow somehow the idea of "semantic versioning" so the format is `x.y.z` being `x` for major releases (breaking changes on the api), `y` is for features (that won't break the api like add a property) and `z` for bugfixes. 

This one is a special case since it's a fork from an external repository, luckily I think they use the same versioning format than we!